pkg/version/version.go

```diff
@@ -38,7 +38,7 @@ var (
 	// It is customary to bump the version by incrementing the numeral after
 	// the `dev` tag. The SQL migration script name must correspond to the /new/ version.
 	Promscale                           = "0.4.2-dev.0"
-	PromMigrator                        = "0.0.1"
+	PromMigrator                        = "0.0.2-beta.0"
```

Comment:
why beta?

Reply:
I wasn't sure of this. I think we should let users try retrying before we mark it stable.

Reply:
do you think there is anything in this PR that can break existing functionality, if so, what?

Reply:
There isn't anything breaking I think. The part of retrying when read times-out is not deterministic. More importantly, I think it will be safer to keep the first release of retrying as beta so that we can know about issues (if they exists as corner cases), than calling it **stable**.